wbooks-api/Gemfile

```diff
@@ -63,4 +63,8 @@ end
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 # Flexible authentication solution for Rails with Warden
-gem 'devise', '~> 4.4', '>= 4.4.3'
+gem 'devise', '~> 4.4'
```

Comment:
If you're requiring devise_token_auth you shouldn't need the devise gem here (it'll require it on its own). It's not important though